example/seed-crds/10-crd-extensions.gardener.cloud_operatingsystemconfigs.yaml

```
@@ -336,6 +336,7 @@ spec:
                 description: |-
                   CloudConfig is a structure for containing the generated output for the given operating system
                   config spec. It contains a reference to a secret as the result may contain confidential data.
+                  After Gardener v1.112, this will be only set of OperatingSystemConfigs with purpose 'provision'.
```

This sentence feels incomplete to me. Was it supposed to be like this?
```suggestion
                  After Gardener v1.112, this will be only set for OperatingSystemConfigs with purpose 'provision'.
```